index.js

```diff
@@ -16,17 +16,27 @@ module.exports = () => {
           return;
         }
 
-        let resolvedPath = path.resolve(
-          node.params.slice(
-            `'"`.includes(node.params[0]) ? 1 : 0,
-            `'"`.includes(node.params[~-node.params.length]) ? -1 : undefined
-          )
-        );
+        let id = node.params
+          .replace(/^(url\()?['"]?/, "")
+          .replace(/['"]?(\))?$/, "");
+
         let replacement;
         try {
+          let resolvedPath = resolve.sync(id, {
+            basedir: process.cwd(),
+            moduleDirectory: ["web_modules", "node_modules"],
+            extensions: [".css"],
+            packageFilter: (pkg) => {
+              if (pkg.style) pkg.main = pkg.style;
```

Comment:
Hm, this line (and one below) is not unit-tested... I've never seen the `style` exports in a package before, what are those? Is it a web modules thing?

Reply:
The style name from package.json is commonly used to make it easier to import stylesheets, as a user you don't need to care about where the CSS is located in a package but leave this to the author. Example for this package https://github.com/csstools/sanitize.css/blob/main/package.json#L15 I can:
```
@import url("sanitize.css");
```
And do not have to know the full path:
```
@import url("sanitize.css/sanitize.css");
```

The logic is copy-paste from postcss-import. It checks for style and uses that before main (if it ends with .css), or falling back to index.css

I will try to look in to how to test this when I am back from vacation 👍